docs/getting_started/explorer.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Explorer
+sidebar_position: 6
+---
+
+From your  Calimero [Console](https://app.calimero.network/dashboard) you can access the explorer which provides you with the ability to view all crypto transactions and information including: accounts, blocks, transactions and receipts, Fungible Tokens and Non-Fungible Tokens in details. 
+
+It is by default a fully private explorer only available to the administrators of the network, but specific information can be shared via API endpoints or shared links through the permissions management service implemented inside the Console.
```

Comment:
Not only to administrators, but to everybody with access to the console and the shard.